config/label_sync/labels.yaml

```diff
@@ -90,6 +90,21 @@ default:
       prowPlugin: label
       addedBy: humans
 
+    # Specific labels for issue triage
+    # See https://apenwarr.ca/log/20171213#slide32 for why we want to have these labels
+    - color: 47d29b
+      description: Issues which should be fixed (post-triage)
+      name: triage/accepted
+      target: issues
+      prowPlugin: label
+      addedby: humans
+    - color: 47d29b
```

Comment:
I intentionally used the same color for both, as the goal here is to use these to find issues lacking these two labels and assign one of the labels or close the issue.